Python/pystate.c

```diff
@@ -452,6 +453,7 @@ init_runtime(_PyRuntimeState *runtime,
     runtime->unicode_state.ids.next_index = unicode_next_index;
 
     runtime->_initialized = 1;
+    _Py_EmscriptenTrampoline_Init(runtime);
```

Comment:
I'd expect `_initialized = 1` to be the last thing happening in this function, so I'd consider moving `_Py_EmscriptenTrampoline_Init` up a line or two.